03week/airplane/css/style.css

```diff
@@ -0,0 +1,130 @@
+body {
+  background-image: url("../img/airline.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin: 0 auto;
+}
+
+.header-logo {
+  display: inline-block;
+  background: url("../img/logo.png");
+  background-repeat: no-repeat;
+  background-color: #093755;
+  border-bottom: 1px solid #fff;
+  color: #fff;
+  width: 100%;
+  height: 81px;
+  font-family: sans-serif;
+  font-size: 14px;
+}
+
+.Navlinks {
+  display: inline;
+  float: right;
+  padding-right: 10%;
+  padding-top: 30px;
+  cursor: pointer;
+  <
+}
+
+ul {
+  list-style-type: none;
+  text-decoration: none;
+}
+
+li {
+  display: inline;
+  margin: 10px;
+}
+
+a:hover {
+  color: cyan;
+}
+
+.booking-container {
+  background-color: #00007f;
+  width: 400px;
+  height: 375px;
+  margin-left: 175px;
+  margin-top: 60px;
+}
+
+.booking-options {
+  background-color: teal;
+  height: 50px;
+}
+
+.booking-options div {
+  display: inline-block;
+  width: 30%;
+  text-align: center;
+}
+
+.booking-options .active {
+  background: blue;
+}
+
+.table {
+  border-collapse: collapse;
+  border: none;
+}
+
+.input-1 {
```

Comment:
Notice how `.input-1` and `.input-2` share most of their properties. Think how you can group them together into a single CSS rule.